src/frontend/Library.mli

```diff
@@ -0,0 +1,41 @@
+(** This is a {{:https://dune.readthedocs.io/en/latest/variants.html}virtual module}
+    which is filled in at link time with a module specifying a backend-specific
+    Stan library. *)
+
+open Middle
+open Core_kernel
+open Std_library_utils
+
+val function_signatures : (string, signature list) Hashtbl.t
+(** Mapping from names to signature(s) of functions *)
+
+val distribution_families : string list
```

Comment:
Should we have docs for all of these if this is a layer people are going to be integrating with? Like just to say, "Here is what the intent of this function is / here is how it is used."

Reply:
Agreed. Will add more comments